src/MainView.js

```diff
@@ -75,7 +75,7 @@ type Props = {
   onResetCategory: (() => void),
   onClickSearchToolbar: (() => void),
   onCloseSearchToolbar: (() => void),
-  onClickSearchButton: (() => void),
+  onClickSearchButton: ((event: UIEvent) => void),
```

Comment:
Is the event even still required? I cannot find anything in this change. Its weird that one method is different, and also weird that the internal implementation details bleed out of the component

Reply:
Yeah, <App /> uses it for `stopPropagation`. Probably it's better to stop the propagation inside the component, you are right.